spec/lss_loader_spec.rb

```diff
@@ -121,6 +121,12 @@
     expect_basic_hierarchy
   end
 
+  it "ignores advice locations which are marked as excluded" do
```

Comment:
Would it be useful to have individual tests for each of the three states that mark an advice location as 'excluded' ?

Reply:
I think the first one is covered by another test (checks for data integrity), but the latter 2 could be separate, but I guess there's a discussion about how many cases a test should cover, every combination etc

Reply:
Gotcha. I tend to agree about having a minimal amount of tests, but in the case where the logic is ensuring that a future change won't risk excluded locations from showing up on the public website, I tend to err on the side of caution.